e2e/bitrise.yml

```diff
@@ -0,0 +1,201 @@
+format_version: 11
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - ORIG_BITRISE_SOURCE_DIR: $BITRISE_SOURCE_DIR
+
+workflows:
+  test_objc:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
```

Comment:
I suggest moving shared env vars into app env vars, so that you can easier review, which test case sets which step inputs: https://github.com/bitrise-steplib/steps-xcode-archive/pull/230/files#diff-6505b061ae185d6fd80374148e0e39774b48ebf7aae5ccbfe2931428da50f814R4-R18

Reply:
Extracted XCODE_OUTPUT_TOOL